static/src/js/components/Banner/Banner.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import classNames from 'classnames'
-import { FaTimesCircle } from 'react-icons/fa'
+import { Close } from '@edsc/earthdata-react-icons/horizon-design-system/hds/ui'
+import { AlertHighPriority } from '@edsc/earthdata-react-icons/horizon-design-system/earthdata/ui'
```

Comment:
![image](https://github.com/user-attachments/assets/61b024fd-712b-496a-80a1-d9877e172d18)
This one?